src/welcomePage.ts

```diff
@@ -62,7 +62,11 @@ export class WelcomePage {
     }
 
     private needToShow() {
-        const show = this.context.globalState.get(Constants.ShowWelcomePageAfterUpdating);
+        let show = this.context.globalState.get(Constants.ShowWelcomePageAfterUpdating);
+        if (show === undefined) {
```

Comment:
in the if (show) block, if version is undefined, shall we show the page?

Reply:
if version is undefined, it means user install our extension for the first time, so the welcome page will not show